dev/DemoApp1/plugin/src/lib.rs

```diff
@@ -18,6 +21,15 @@ impl Intf for Component {
     fn helloworld2() -> Result<String, CommonTypes::Error> {
```

Comment:
you could also delete helloworld and helloworld2 if you want, just to clean up the example app